lib/scrabble.rb

```diff
@@ -1,9 +1,18 @@
 class Scrabble
 
   def score(word)
-    1
+    return 0 if word.nil?
+    word_score(word)
   end
 
+  def word_score(word)
+    num_score = word.split("").map do |letter|
+      point_values[letter.upcase]
+    end
+    num_score.sum
+  end
```

Comment:
This all feels nice and compact. Thinking is very clear.

Might consider breaking into even smaller methods, but that would be based on preference. Benefit would be that it allowed you to have more unit testing.